pytorch_lightning/strategies/bagua.py

```diff
@@ -58,7 +57,7 @@ def __init__(self, pl_module: "pl.LightningModule") -> None:
 
 
 class BaguaStrategy(DDPStrategy):
-    distributed_backend = _StrategyType.BAGUA
+    strategy_name = "bagua"
```

Comment:
Should this be just `name`? to avoid the redundancy of `trainer.strategy.strategy_name`